packages/web/src/scss/components/TextField/README.md

```diff
@@ -132,10 +132,29 @@ forget to change not only the input type but also `aria-pressed` and
 ## Validation States
 
 Validation states can be presented either by adding a CSS modifier class
-(`TextField--success`, `TextField--warning`, `TextField--error`), or by adding
+(`TextField--success`, `TextField--warning`, `TextField--danger`, `TextField--error`), or by adding
 a JS interaction class when controlled by JavaScript (`has-success`,
 `has-warning`, `has-error`).
 
+```html
+<div class="TextField TextField--danger">
```

Comment:
we should introduce `has-danger` as well. and deprecate `has-error`.

Reply:
done